CONTRIBUTING.md

```diff
@@ -25,11 +25,12 @@ following installed:
 
 * [Python](https://www.python.org/downloads/)
     - If you're on Windows (Not Windows Subsystem for Linux)
-        - Install Python 2.7 x64
-        - Install at least one Python 3.x x64 - 3.4, 3.5 or 3.6 will all work
+        - Install at least one Python 3.x x64 - anything between 3.4 and 3.8 should work
         - Optional - 32 bit versions
         - Ensure pip support is installed for all versions
         - Install paths can either be in the appropriate Program Files for the bitness, or c:\pythonXY for 64 bit and c:\pythonXY-32 for 32 bit
+    - Add ``python Install Path`` and ``python Install Path``\Scripts to Windows path
+    - Copy ``python Install Path``\python.exe to ``python Install Path``\python3.exe
 
 * [GNU Make](https://www.gnu.org/software/make/)
```

Comment:
```suggestion
* [GNU Make](https://www.gnu.org/software/make/)
    - Install and enable [Windows Subsystem for Linux](https://msdn.microsoft.com/en-us/commandline/wsl/install_guide)
    - Install make
            sudo apt-get install make
```

I don't think the multi-line suggestion is working (maybe I'm doing something wrong). But essentially no longer make a distinction between Windows 10 and older and remove instructions for older.

Reply:
We didn't have separate instructions for older. We had only 3 instructions: for WSL, Windows and MacOS. But I'll clean it up a bit and also remove the references to the older versions of Windows